__tests__/components/Calculator/Calculator.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Calculator from '../../../src/components/Calculator/Calculator'
+
+const setup = (setupProps = {}) => {
+  const defaultProps = {
+    maxScoreWrapper: { primary: { maxScore: { value: 182 } }, lingual: { maxScore: { value: 192 } } },
+    isScoreTableOpened: false,
+    isAnySubjectPresent: false,
+    onToggleClick: jest.fn()
+  }
+  const props = { ...defaultProps, ...setupProps }
+  const wrapper = shallow(<Calculator {...props} />)
+
+  return {
+    props,
+    wrapper
+  }
+}
+
+describe('Calculator component', () => {
+  test('should render', () => {
+    const { wrapper } = setup()
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should render content when isAnySubjectPresent is true', () => {
+    const { wrapper } = setup({ isAnySubjectPresent: true })
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should call onToggleClick when clicked on a button', () => {
+    const { props, wrapper } = setup({ isAnySubjectPresent: true })
+
+    wrapper.find('button').simulate('click')
+    expect(props.onToggleClick).toBeCalled()
+  })
+
+  test('should render the scoreTable when isScoreTableOpened is true', () => {
+    const { wrapper } = setup({ isAnySubjectPresent: true, isScoreTableOpened: true })
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should render button with corresponding name when scoreTable is opened', () => {
+    let { wrapper } = setup({ isAnySubjectPresent: true, isScoreTableOpened: false })
+    let buttonWrapper = wrapper.find('button').shallow()
+
+    expect(buttonWrapper.text()).toEqual('Pokaż tabelę wyników')
+
+    ;({ wrapper } = setup({ isAnySubjectPresent: true, isScoreTableOpened: true }))
```

Comment:
What about splitting this into two separate specs, or putting `;` at the end of 52? As `;` in the beginning of the line is some exotic practice ;)

Here and in other files.

Reply:
I've been doing as you proposed, but "Prettier" corrected it to the form you see. I thought it is a common pattern. I know if I've been using semicolons everywhere, I wouldn't have had this dilemma. 😃

Reply:
Mhm, that's strange pattern that Prettier used, but it kind of makes sense, as it does not use comma, then as exception it makes that visible.